api/account.go

```diff
@@ -1,20 +1,21 @@
 package api
 
 import (
+	"database/sql"
 	db "github.com/Petatron/bank-simulator-backend/db/sqlc"
 	"github.com/gin-gonic/gin"
 	"net/http"
 )
 
 // createAccountRequest defines the body for createAccount API request
-type createAccount struct {
+type createAccountRequest struct {
 	Owner    string `json:"owner" binding:"required"`
-	Currency string `json:"currency" binding:"required, oneof=USD EUR CAD"`
+	Currency string `json:"currency" binding:"required,oneof=USD EUR CAD"`
```

Comment:
Nit: Instead of hard-coding support currency here, can we create a constant list to include all supported currencies for easier maintenance?

Reply:
Good point. Will create a ticket for this.